setup.py

```diff
@@ -67,6 +67,7 @@ def get_snakemake_pkg(extras=""):
     "mock>=3.0,<4",
     "PyYAML>=5.1,<6.0",
     "Werkzeug>=0.14.1",
+    "wcmatch",
```

Comment:
It seems that wcmatch 8.3 is the last Python 3.6 compatible release, so we could in theory do: 

```python
"wcmatch>=8.3,<8.5",
```

Although it seems  better to be more precise here, something like:

```python
"wcmatch>=8.3<8.4 ; python_version<'3.7'",
"wcmatch>=8.4.1,<8.5 ; python_version>='3.7'",
```

P.S. The example going somewhat "conservatively" for the upper boundary of 8.5 instead of 9.0 as per wcmatch release history regarding features vs bug fixes.

Reply:
pip automatically install the last compatible version, so using `wcmatch>=8.3<8.5` should be good for all the Python versions we support

Reply:
I guess it would be OK to use say `<9.0` for the upper boundary, provided that we _always cut_ the appropriate version for the r-server component. Since we don't use `**` on the upload side (yet), even if r-client and r-server differ somehow, it should be quite OK.

Another solution would be to add the wcmatch dependency to the "kubernetes" installation target, if these functions are really only used on the r-server workspace side for things for now.